scripts/start_logging.sh

```diff
@@ -3,6 +3,17 @@
 # path to log file - global variable
 FILE="$1"
 
+# If called directly, read context and enable logging
```

Comment:
IMO this belongs better in toggle_logging.sh as that's the "user callable" file and this one is a lower level function, but that level of re-org would make it hard to follow the PR as well as new naming conventions, which are better left to the owner.